tests/PgSqlServerTest/PgSqlServerTest.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+# vim: set ai et ts=4 sw=4:
+
+import os
+import random
+import subprocess
+import requests
+import logging
+import time
+import socket
+import sys
+import psycopg2
+
+START = os.environ.get('HURMADB_PORT') is None
+PORT = int(os.getenv('HURMADB_PORT', 5432))
+con = None
+logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
+
+def hurmadb_start(port):
+    """
+    Start HurmaDB on given port, return an object that corresponds to the created process.
+    """
+    if START:
+        return subprocess.Popen(['../../hurmadb', str(port)])
+    else:
+        return None
+
+def hurmadb_stop(pobj):
+    """
+    Stop HurmaDB process created by hurmadb_start()
+    """
+    if con:
+        con.close()
+
+    if START:
+        pobj.wait()
+
+class TestBasic:
+    def setup_class(self):
+        self.log = logging.getLogger('HurmaDB')
+        self.pobj = hurmadb_start(PORT)
+        time.sleep(3) # Give RocksDB some time to initialize
+        self.log.debug("HurmaDB started on port {}".format(PORT))
+
+    def teardown_class(self):
+        hurmadb_stop(self.pobj)
+        self.log.debug("HurmaDB stopped")
+
+    def test_connection(self):
+        try:
+            self.log.debug("Running test_connection")
+            con = psycopg2.connect("dbname='postgres' user='nikitos' host='localhost' password='pass'")             
+            
+            self.log.debug("Running test_connection")
+            cur.execute("UPDATE Cars SET Name='Alfred Schmidt' WHERE Id=1121;")
+            cur.execute("DELETE FROM Cars where id = 1120")
+
+        except psycopg2.DatabaseError, e:
```

Comment:
This syntax is not allowed in Python 3:
```
E     File "/home/eax/projects/c/hurmadb/tests/PgSqlServerTest/PgSqlServerTest.py", line 59
E       except psycopg2.DatabaseError, e:
E                                    ^
E   SyntaxError: invalid syntax
```

The code should work with Python 3 since Python 2 will be officially deprecated in 2 years, in 2020.